docs/deployment.md

```diff
@@ -132,6 +133,49 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
 
         $ now scale https://a-fancier-url.now.sh 1
 
+### GitHub Actions
+
+> **Heads Up!** [GitHub Actions](https://github.com/features/actions) is still in limited beta.
+
+GitHub Actions allows you to trigger workflows based on GitHub events, which makes it a great fit for running Probot Apps. To run a your app on GitHub Actions:
+
+1. Add a `Dockerfile` to your app:
+    ```
+    FROM node:10
+
+    ENV PATH=$PATH:/app/node_modules/.bin
+    WORKDIR /app
+    COPY . .
+    RUN npm install --production
+
+    ENTRYPOINT ["probot", "receive"]
+    CMD ["/app/index.js"]
+    ```
+
+1. In the repository that you want to run the app, create a `.github/main.workflow` file that defines the action and listens for any events that your app depends on. For example, here is the workflow for @jasonetco's [TODO](https://github.com/jasonetco/todo):
+    ```
+    workflow "Check for TODOs in Pull Requests" {
+      on = "pull_request"
+      resolves = "TODO"
+    }
+
+    workflow "Check for TODOs on Push" {
+      on = "push"
+      resolves = "TODO"
+    }
+
+    action "TODO" {
+      uses = "jasonetco/todo"
+      secrets = ["GITHUB_TOKEN"]
+    }
+    ```
+
+There are a few caveats when running Probot Apps on GitHub Actions:
+
+- The GitHub API token available to actions has a [fixed set of permisisons](https://developer.github.com/actions/creating-workflows/storing-secrets/#github-token-secret), and only has access to the repository that triggered the action. `robot.auth()` will always return a GitHub client authenticated for the current repository.
```

Comment:
Can we make this `app.auth()` and link to: https://probot.github.io/api/latest/classes/application.html#auth